Added Projects Section

vikytech opened the pull request:
Added ability to render **Projects** section

vikytech commented:
@mudassir0909  / Any other contributor
Can we please have a look into this PR

mudassir0909 commented:
@vikytech hey, thanks for the pull request but is the `projects` field part of the official JSON resume schema? I couldn't find it [here](https://jsonresume.org/schema/)

mudassir0909 commented:
Please ignore my previous comment, looks like it is part of the official schema as [per Github](https://github.com/jsonresume/resume-schema/blob/v1.0.0/schema.json)

Few comments:
* Please update the `resume-schema` version inside the `package.json` to the latest version `0.1.3` 
* You also need to update this line in `serve.js` from 

https://github.com/mudassir0909/jsonresume-theme-elegant/blob/7d4262401132931f56919de94441fa864ba45333/serve.js#L11

to 

```
const resume = JSON.parse(fs.readFileSync('node_modules/resume-schema/sample.resume.json', 'utf8'));
```

With these changes in place, you should be able to see the projects when you do `grunt exec:run_server`. When I ran it seems to be missing the project name

<img width="709" alt="Richard Hendriks 2020-09-14 19-16-21" src="https://user-images.githubusercontent.com/1861842/93068088-5f859d80-f6bf-11ea-95c8-2fcf86619dd3.png">

I think with these changes in place we should be able to push forward to getting this PR merged very soon :)

mudassir0909 commented on index.js:
```diff
@@ -36,6 +36,30 @@ function render(resume) {
+          project_info.endDate = utils.getFormattedDate(end_date);
+        }
+
+        project_info.name = convertMarkdown(project_info.name);
```
IMO we shouldn't allow markdown in the project's name. Visually, it should follow the styles specified by the theme

vikytech replied:
Sure, Makes sense

vikytech commented:
@mudassir0909 Thanks for the reply, I will update the changes ASAP

vikytech commented:
**Updates:**

1) Updated the resume-schema version to `0.1.3`
2) Fixed Project name rendring.
![image](https://user-images.githubusercontent.com/3875111/93326155-03915500-f836-11ea-9e2f-a7dd680917d0.png)

@mudassir0909 let me know if any changes to be done.
